README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
Firstly I moved these two together to client, but server side will have problem, like simulator_service and moveit crassed since they can't connect to action server.

After that I only move industrial_robot_simulator to client side, then the delay problem shows up.

[ERROR] [1460485259.724249914]: MoveitSimpleControllerManager: Action client not connected: joint_trajectory_action
[ERROR] [1460485246.466222137]: Could not connect to action: 'joint_trajectory_action'
[chencheng/simulator_service-8] process has died [pid 20016, exit code -6, cmd /home/chencheng/catkin_ws/devel/lib/industrial_robot_simulator_service/simulator_service_node controller_joint_names:=/controller_joint_names __name:=simulator_service __log:=/home/chencheng/.ros/log/42401eb4-00db-11e6-8ce7-c0cb38142cc0/chencheng-simulator_service-8.log].
log file: /home/chencheng/.ros/log/42401eb4-00db-11e6-8ce7-c0cb38142cc0/chencheng-simulator_service-8*.log